node/pkg/db/db.go

```diff
@@ -7,9 +7,17 @@ import (
 	"strings"
 
 	"github.com/dgraph-io/badger/v3"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/wormhole-foundation/wormhole/sdk/vaa"
 )
 
+var storedVaaTotal = promauto.NewCounter(
+	prometheus.CounterOpts{
+		Name: "wormhole_db_vaa_ctr",
```

Comment:
Would it be possible to give this a better name? Maybe `wormhole_db_total_vaas`?

Reply:
done